index.html

```diff
@@ -2,6 +2,23 @@
 layout: default
 title: Repose
 ---
+<nav class="top-bar" data-topbar role="navigation">
```

Comment:
It's not. I tried doing that, but it broke the menu functionality, and it wasn't important enough to try to fix